modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/Andes.java

```diff
@@ -165,6 +169,10 @@ public void startSafeZoneAnalysisWorker() {
      */
     public void messageReceived(AndesMessage message, AndesChannel andesChannel, PubAckHandler pubAckHandler) {
         inboundEventManager.messageReceived(message, andesChannel, pubAckHandler);
+
+        //Adding metrics meter for message rate
+        Meter messageMeter = MetricManager.meter(Level.INFO, this.getClass() + MetricsConstants.MSG_RECEIVE_RATE);
```

Comment:
Carbon Metrics API associate a Level with each Metric. These levels are similar to the Levels used in Logging Libraries. When we add a metric to metric manager, we need to provide level for that particular metric. Later we can set the level which we need to enable metrics.